monkey/tests/utils.py

```diff
@@ -31,3 +31,15 @@ def add_files_to_dir(parent_dir: Path, file_names: Iterable[str]) -> Iterable[Pa
         f.touch()
 
     return files
+
+
+# This is only needed since values are compared in configuration objects in the tests.
+# In practice, the list/tuple differences shouldn't make any difference since both are iterable.
+def convert_lists_to_tuples(configuration: Mapping):
```

Comment:
Lets find a better name for this function. By the name I expect bunch of lists to be converted to tuples, not a mapping which may contain lists or mapping to be converted to tuples and recursively to be checked.

Reply:
How about now?